### PR TITLE
[SM-576] Rainbow kit

### PR DESCRIPTION
**Pull request by Asfiroth**
# Proposed Changes

### CODE REVIEW

File: app-login/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
+    <!-- <link href="https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.css" rel="stylesheet">
```
Comment (JamesReate) [inline app-login/index.html:8]: remove commented out code

File: app-login/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  challenge?: string;
+  constructor(param: Partial<DIMODexMessage>) {
+    this.state = param.state;
+    this.challenge = param.challenge;
+  }
+};
+
+const performTokenExchange = async () => {
+  try {
+    const response = await fetch('http://localhost:3003/api/token_exchange', {
```
Comment (JamesReate) [inline app-login/src/App.tsx:21]: make the http://localhost:3003 configurable so easy to switch from local development to docker release.

File: app-login/src/vite-env.d.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/// <reference types="vite/client" />
+interface ImportMetaEnv {
+    readonly DIMO_API_BASEURL: string;
```
Comment (JamesReate) [inline app-login/src/vite-env.d.ts:3]: how / where is this configured?

Comment (Asfiroth) [reply app-login/src/vite-env.d.ts:3]: this is in the Docker file

File: app-login/src/App.tsx (hunk 2 above)
Comment (Asfiroth) [reply app-login/src/App.tsx:21]: I think it's better to remove that function since it's not used

**Review by JamesReate — APPROVED** [review]

**Pull request merged**